api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -515,7 +515,7 @@ fun ApiScannerJobConfiguration.mapToModel() = ScannerJobConfiguration(
 fun Secret.mapToApi() = ApiSecret(name, description)
 
 fun VulnerabilityWithIdentifier.mapToApi() =
-    ApiVulnerabilityWithIdentifier(vulnerability.mapToApi(), identifier.mapToApi())
+    ApiVulnerabilityWithIdentifier(vulnerability.mapToApi(), identifier.mapToApi(), rating.mapToApi())
```

Comment:
Commit message nit: The wrapping seems to be too narrow, i.e. you wrap at column 55 (?) instead of 75.